climpred/predictability_horizon.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import xarray as xr
+
+
+def _last_item_cond_true(cond, dim, shift=-1):
+    """Get the last item of condition ``cond`` which is True.
+
+    Args:
+        cond (xr.DataArray, xr.Dataset): Condition True means predictable. ``cond``
+            contains at least the dimension lead and checks for how many leads the
+            ``condition`` is true and hence skill is predictable.
```

Comment:
cond (xr.DataArray, xr.Dataset): User-defined boolean array where True means the system is predictable at the given lead. E.g., this could be based on the dynamical forecast beating a reference forecast, p values, confidence intervals, etc. ``cond`` should contain the dimension ``lead`` at the minimum.

Reply:
Just trying to make the descriptions a little more clear!